models/articles/models.py

```diff
@@ -2,6 +2,7 @@
 
 
 class Article(db.Model):
+    __tablename__ = 'article'
```

Comment:
Please rename tables to be plural name.
For example:
article -> articles
category - > categories